client/keys.go

```diff
@@ -50,12 +56,15 @@ The pass backend requires GnuPG: https://gnupg.org/
 	// support adding Ethereum supported keys
 	addCmd := keys.AddKeyCommand()
 
-	// update the default signing algorithm value to "eth_secp256k1"
-	algoFlag := addCmd.Flag(flags.FlagKeyType)
-	algoFlag.DefValue = string(hd.EthSecp256k1Type)
-	err := algoFlag.Value.Set(string(hd.EthSecp256k1Type))
-	if err != nil {
-		panic(err)
+	ethSecp256k1Str := string(hd.EthSecp256k1Type)
+	if defaultKeyType == ethSecp256k1Str {
```

Comment:
Why not use directly the `PubKeyType`  instead of `string`? Now you convert this type for the input and then convert the `hd.EthSefcp256k1Type` to match the string.